Header header
geometry_msgs/Pose center
geometry_msgs/PoseStamped[] path 
geometry_msgs/Vector3  obs_box 
bool is_static
uint32 label
